__tests__/usecases/rules_priority.json

```diff
@@ -0,0 +1,78 @@
+{
```

Comment:
Not necessary, but usually a good idea to run things like this through prettier for consistency

Reply:
Done